src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonShadowAssassinNotification.kt

```diff
@@ -0,0 +1,33 @@
+package at.hannibal2.skyhanni.features.dungeon
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.TitleManager
+import at.hannibal2.skyhanni.events.PacketEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.SoundUtils
+import net.minecraft.network.play.server.S44PacketWorldBorder
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.lang.reflect.Field
+import kotlin.time.Duration.Companion.seconds
+
+class DungeonShadowAssassinNotification {
+    @SubscribeEvent
+    fun onWorldBoarderChange(event: PacketEvent.ReceiveEvent) {
+        if (!LorenzUtils.inSkyBlock || !LorenzUtils.inDungeons) return
+        if (!SkyHanniMod.feature.dungeon.shadowAssassinJumpNotifier) return
+        //F7 and maybe newer, currently not released, floors have shadow assassins, so they are allowed to be listed here
+        val disable = if (DungeonAPI.dungeonFloor == null) true else DungeonAPI.dungeonFloor!!.contains("7")
+        if (DungeonAPI.inBossRoom && disable) return
+        if (event.packet !is S44PacketWorldBorder) return
+        val packet: S44PacketWorldBorder = event.packet
+        //Did not find another way to read the packet :/
+        val action: Field = packet.javaClass.getDeclaredField("field_179795_a")
+        action.isAccessible = true
+        val warningTime: Field = packet.javaClass.getDeclaredField("field_179796_h")
+        warningTime.isAccessible = true
```

Comment:
When using reflection, use `ReflectionHelper.findField` with all names for that field to allow running in the dev env as well as the standard obfuscated environment. Also don't use reflection here at all, instead use a [Accessor mixin](https://github.com/hannibal002/SkyHanni/blob/beta/src/main/java/at/hannibal2/skyhanni/mixins/transformers/AccessorChatComponentText.java).